starter-code/js/index.js

```diff
@@ -11,7 +11,33 @@ function updatePriceByProduct(productPrice, index){
 }
 
 function getTotalPrice() {
+  var totalCost = 0;
+  for (var x = 0; x < document.querySelectorAll('.product').length ; x ++) {
 
+  var unitValue = Number(document.querySelectorAll('.cost')[x].innerHTML);
+  var totalUnits = Number(document.querySelectorAll('input')[x].value);
```

Comment:
Same here, create an array outside the for and iterate using this array instead of searching in the document.